fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -179,34 +168,30 @@
     private static final Set<Property> ldpManagedProperties = of(CONTAINS);
 
     // REPOSITORY INFORMATION
-    public static final Property HAS_OBJECT_COUNT =
+    private static final Property HAS_OBJECT_COUNT =
             createProperty(REPOSITORY_NAMESPACE + "objectCount");
-    public static final Property HAS_OBJECT_SIZE =
+    private static final Property HAS_OBJECT_SIZE =
             createProperty(REPOSITORY_NAMESPACE + "objectSize");
     public static final Property HAS_TRANSACTION_SERVICE =
             createProperty(REPOSITORY_NAMESPACE + "hasTransactionProvider");
-    public static final Property HAS_ACCESS_ROLES_SERVICE =
-            createProperty(REPOSITORY_NAMESPACE + "hasAccessRoles");
 
-    public static final Set<Property> repositoryProperties = of(
+    private static final Set<Property> repositoryProperties = of(
             HAS_OBJECT_COUNT, HAS_OBJECT_SIZE, HAS_TRANSACTION_SERVICE);
 
     // NAMESPACES
-    public static final Property HAS_NAMESPACE_PREFIX =
+    private static final Property HAS_NAMESPACE_PREFIX =
```

Comment:
`HAS_NAMESPACE_URI` and `HAS_NAMESPACE_PREFIX` do not appear to be used for anything but specifying that they are SMTs, perhaps we should remove them?